packages/hash/frontend/src/blocks/page/Comments/CommentTextField.tsx

```diff
@@ -182,7 +163,7 @@ export const CommentTextField: FunctionComponent<CommentTextFieldProps> = ({
       />
 
       <Box display="flex" alignItems="flex-end" margin={1.5}>
-        {loading ? (
+        {loadingRef.current ? (
```

Comment:
I missed that `loading` is used in the UI. I'm sorry, that's my bad. It needs to be state, not a ref, because writing to refs doesn't trigger a re-render and therefore reading from them during render is unsafe.

Reply:
Changed back loading to state in 5f28ce250602f61a8dbc58f6c68755aa47edf71f